.github/workflows/ruby.yml

```diff
@@ -0,0 +1,117 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+name: Ruby
+
+on:
+  push:
+    branches: [ production ]
+  pull_request:
+    branches: [ production ]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    services:
+      db:
+        image: postgres:11
```

Comment:
Can change to 9.5? We're still using 9.5 in our server and it'll be great to match.

Reply:
I tried it and it will summon such an error like:

ERROR:  unrecognized configuration parameter "idle_in_transaction_session_timeout"

Found same problem at this link:
https://stackoverflow.com/questions/41528168/postgresql-unrecognized-configuration-parameter

But, i dont get it. I confuse about the solution that it explained.

Reply:
Might be to do with the fact that the postgres client installed is too new. But that's alright, let us just keep this at 11 then.